src/tests/end-to-end/common/browser.ts

```diff
@@ -89,6 +89,11 @@ export class Browser {
         return page;
     }
 
+    public async gotoContentPage(existingPage: ContentPage, newContentPath: string): Promise<void> {
```

Comment:
Not a high priority comment here: this seems like a static function... I'm wondering if that's desirable or not... or maybe we can move this to the `ContentPage` class (if it exist anyway)

Reply:
It's not quite static because forming the URL depends on the background page to identify the extension URL origin